### PR TITLE
Add SVD files for ATSAMS70 & ATSAMV70

### PR DESCRIPTION
**Pull request by michalfita**
The [atsams70-rust](https://github.com/michalfita/atsams70-rust) project requires these missing SVD files.

### CODE REVIEW

**Review by posborne — APPROVED** [review]
Thanks @michalfita, I'll merge once CI completes now that I approved the run.

**Comment by tmplt** [comment]
@posborne, CI has completed.

**Pull request merged**
